src/rules/variableNameRule.ts

```diff
@@ -31,6 +31,7 @@ const OPTION_LEADING_UNDERSCORE = "allow-leading-underscore";
 const OPTION_TRAILING_UNDERSCORE = "allow-trailing-underscore";
 const OPTION_BAN_KEYWORDS = "ban-keywords";
 const OPTION_CHECK_FORMAT = "check-format";
+const OPTION_ALL_CAPS_FOR_CONST = "all-caps-for-const";
```

Comment:
I think something like `"const-only-for-caps" would actually be more accurate? The way this is worded makes it seem like `const`s must be in all caps.